build-tests/x86/fedora/test-image-live-disk/appliance.kiwi

```diff
@@ -30,7 +30,12 @@
         </type>
     </preferences>
     <preferences profiles="Virtual">
-        <type image="oem" filesystem="ext4" kernelcmdline="console=ttyS0" firmware="uefi" format="qcow2">
+        <type image="oem" filesystem="btrfs" kernelcmdline="console=ttyS0" firmware="uefi" format="qcow2" btrfs_create_toplevel_subvolume="true">
+            <systemdisk name="fedora" rootfs_label="fedora">
+                <volume name="@root=root"/>
+                <volume name="home" mountpoint="/home" parent="/"/>
```

Comment:
Will this ensure that `home` is a separate top-level subvolume rather than being nested on the root subvolume?

Reply:
yes

Reply:
parent is '/' for nesting under the root subvolume you wold have to say parent="root" or not specify the parent